### PR TITLE
Use summary.json to controll the overview charts of benchmarks

### PR DESCRIPTION
**Pull request by grief8**
This PR introduces:
1. An overview chart for existed benchmarks. See [this](https://grief8.github.io/benchmark/)
2. `summary.json` for overview chart controlling. See [this](https://grief8.github.io/benchmark/lmbench/) where `[Semaphores] The cost of semop` is not included.

The benchmark will not appear in the overview chart if it is not listed in `summary.json`. But it will still be included in the detailed benchmark results (at the back). The sequence of the benchmarks will be the same as the sequence in the overview chart.

### CODE REVIEW

**Comment by grief8** [comment]
@boterinas claim ready

**Comment by boterinas** [comment]
**Error**: Parsing assign command in [comment](https://github.com/asterinas/asterinas/pull/1376#issuecomment-2372827006) failed: ...' claim' | error: expected end of command at >| ' ready'...

Please file an issue on GitHub at [triagebot](https://github.com/rust-lang/triagebot) if there's a problem with this bot, or reach out on [#t-infra](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra) on Zulip.

File: test/benchmark/lmbench/summary.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "tcp_loopback_select_lat",
+    "tcp_loopback_http_bw",
+    "udp_loopback_lat",
+    "mem_read_bw",
+    "mem_write_bw",
+    "mem_copy_bw",
+    "mem_pagefault_lat",
+    "mem_mmap_bw",
+    "mem_mmap_lat"
+  ]
```
Comment (tatetian) [inline test/benchmark/lmbench/summary.json:42]: I reordered the benchmarks and removed some. Use the one below.

```json
{
  "benchmarks": [
    "mem_read_bw",
    "mem_write_bw",
    "mem_copy_bw",
    "mem_pagefault_lat",
    "mem_mmap_bw",
    "mem_mmap_lat",
    "process_getppid_lat",
    "process_ctx_lat",
    "process_fork_lat",
    "process_exec_lat",
    "process_shell_lat",
    "signal_install_lat",
    "signal_catch_lat",
    "signal_prot_lat",
    "pipe_lat",
    "pipe_bw",
    "fifo_lat",
    "unix_lat",
    "unix_bw",
    "unix_connect_lat",
    "vfs_open_lat",
    "vfs_read_lat",
    "vfs_write_lat",
    "vfs_stat_lat",
    "vfs_fstat_lat",
    "vfs_fcntl_lat",
    "vfs_read_pagecache_bw",
    "vfs_select_lat",
    "ramfs_create_delete_files_0k_ops",
    "ramfs_create_delete_files_10k_ops",
    "ext2_copy_files_bw",
    "tcp_loopback_bw",
    "tcp_loopback_lat",
    "tcp_loopback_connect_lat",
    "tcp_loopback_select_lat",
    "tcp_loopback_http_bw",
    "udp_loopback_lat"
  ]
}
```

Comment (grief8) [reply test/benchmark/lmbench/summary.json:40]: Get it

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**